qiskit/primitives/containers/observable.py

```diff
@@ -0,0 +1,188 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""
+Container class for an Estimator observable.
+"""
+from __future__ import annotations
+
+import re
+from collections.abc import Mapping, Iterable
```

Comment:
Python 3.8 does not seem to support `collections.abc.Mapping[KT, VT]` and `collections.abc.Iterable[T]`.
You might need to use `typing.Mapping` and `typing.Iterable` instead.

Reference: [PEP 585](https://peps.python.org/pep-0585/) – Type Hinting Generics In Standard Collections